charts/node/values.yaml

```diff
@@ -89,14 +89,41 @@ node:
   #podManagementPolicy: Parallel
   #customChainspecUrl:
 
-substrateApiSidecar:
-  enabled: false
+  # Enables Jaeger Agent as a sidecar
+  tracing:
+    enabled: false
+
+  # Enables Sustrate API as a sidecar 
+  substrateApi:
+    enabled: false
+
+# https://github.com/paritytech/substrate-api-sidecar
+substrateApi:
   image:
     repository: parity/substrate-api-sidecar
     tag: latest
   env: {}
   resources: {}
 
+# https://github.com/jaegertracing/jaeger
+jaegerAgent:
+  image:
+    repository: jaegertracing/jaeger-agent
+    tag: 1.28.0
```

Comment:
Let's use `latest` here and not set a default version here as we can't always keep it up date. Also, automated helm security analysis tools will assign a bad score to your chart because it has old version of the docker images. However when using the chart, a versioned tag should always be set.